dnf-plugins-core.spec

```diff
@@ -145,21 +149,28 @@ Summary:    Core Plugins for DNF
 BuildRequires:  python3-dbus
 BuildRequires:  python3-devel
 BuildRequires:  python3-dnf >= %{dnf_lowest_compatible}
+BuildRequires:  python3-systemd
+BuildRequires:  pkgconfig(systemd)
+BuildRequires:  systemd
+%{?systemd_requires}
```

Comment:
This should be changed to `%{?systemd_ordering}` instead, so that we don't have a hard dep on systemd at runtime.

Reply:
Thanks, updated.